### ISSUE TITLE
Add a new map layout for cycling

### ISSUE THREAD

**Issue by Vucumpra** [osmandapp/OsmAnd#20439]
### Describe the idea (required)

Add a new map layout  inspired by  _CylOSM_ layer on openstreetmap.org

### Tell us about the expected behaviour (required)

So, a new layout including cycle way in contrast, where you can see low speed street in contrast, bad surface (sette ou cobbelstone) in contrast, dangerous crossing in contrast, etc...

### Tell us about alternatives you've considered (required)

Alternative is to apply "cycling visual graphic design" on all layout map when "cycling" profil is active.

**Comment by mikehgentry** [comment]
This is quite a bit like CyclOSM

https://github.com/OsmAnd-Rendering/Cycling

**Comment by Vucumpra** [comment]
Yes is +/- "similar". So, what about provide this bike layout in native, without to install it after?
